ReadMe.md

```
@@ -269,13 +269,30 @@ Combine network throttling and CPU slowdown to approximate mobile users on spott
 
-Both `--network` and `--cpu` accept a list. Every combination is raced separately, each into its own results subdirectory, with a top-level `index.html` linking them:
+Both `--network` and `--cpu` accept a list. Every combination is raced separately, each into its own results subdirectory:
 
 ```bash
 node race.js ./races/my-race --cpu=1,4              # cpu1x/, cpu4x/
 node race.js ./races/my-race --network=slow-3g,4g --cpu=1,4  # slow-3g-cpu1x/, slow-3g-cpu4x/, 4g-cpu1x/, 4g-cpu4x/
 ```
 
+Afterwards you get a **performance matrix** — network presets down the side, CPU rates across the top, every racer's total time in each cell — printed to the terminal and rendered as the top-level `index.html`, where each cell links to that condition's own results:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)